Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.10
```

Comment:
nit: 
does it allow to add local package via Xcode 15.2 (it uses swift 5.9)? Or have you succeeded with Xcode 15.3?

Reply:
@AlexLevshenkov 
I was able to accomplish this in the following way, in Xcode 15.3:
1. Downloaded the necessary branch as a zip archive, unzipped it
2. Added this archive to my project
3. Changed the path to the framework, from absolute from `Downloads` to relational in the root folder of my project
Done:)